cli/Valet/Site.php

```diff
@@ -384,6 +384,72 @@ function replaceOldDomainWithNew($siteConf, $old, $new)
         return $siteConf;
     }
 
+    /**
+     * Resecure all currently secured sites with a fresh loopback address.
+     *
+     * @param  string  $oldLoopback
+     * @param  string  $loopback
+     * @return void
+     */
+    function resecureForNewLoopback($oldLoopback, $loopback)
```

Comment:
This function is 95% duplicate of resecureForNewTld().
Perhaps the two functions can be combined?  
The only difference is the call to replaceOldDomainWithNew() and replaceOldLoopbackWithNew(). Would it be sensible to just call both of these subfunctions from within resecureForNewTld() ?